include/deal.II/grid/grid_generator.h

```diff
@@ -105,8 +105,9 @@ namespace GridGenerator
   /**
    * Create a $d$-<a href="https://en.wikipedia.org/wiki/Simplex">simplex</a>
    * (i.e., a triangle in 2d, or a tetrahedron in 3d) with
-   * $d+1$ corners. Since deal.II does not support triangular and
-   * tetrahedral cells, the simplex described by the input arguments
+   * $d+1$ corners. Since this function dates back to a time when deal.II
+   * did not support triangular and tetrahedral cells (which it does now),
+   * the simplex created by this function and described by the input arguments
    * is subdivided into quadrilaterals and hexahedra by adding edge,
    * face, and simplex midpoints, resulting in a mesh that consists of
    * $d+1$ quadrilateral or hexahedral cells.
```

Comment:
Seeing this text and comparing against our current abilities, do you think we should mention the way to create a simplex consisting of a single simplex element? We probably do not need to explain all details, but I would mention both https://github.com/dealii/dealii/blob/8de1a8f7b3440c2d668bbc8290bf91ae97b8b0d0/include/deal.II/grid/grid_generator.h#L137-L145 for an object in the reference coordinates and the https://github.com/dealii/dealii/blob/8de1a8f7b3440c2d668bbc8290bf91ae97b8b0d0/include/deal.II/grid/tria.h#L2057-L2060 function that takes the argument passed in here as well, plus the description of a single cell. Or maybe we have even more elegant ways?

Reply:
Yes, good idea. I've added such text.